operantanalysis/scripts/CRF.py

```diff
@@ -0,0 +1,53 @@
+from operantanalysis import load_file, extract_info_from_file, reward_retrieval, lever_pressing, lever_press_latency
+import glob
+from tkinter import filedialog
+from tkinter import *  # noqa
+import pandas as pd
+import matplotlib
+matplotlib.use("TkAgg")
+from matplotlib import pyplot as plt  # noqa
+
+days = int(input("How many days would you like to analyze?"))
+column_list = ['Subject', 'Day', 'Dippers', 'Dippers Retrieved', 'Retrieval Latency',
+               'Lever Presses', 'Lever Press Latency']
+df = pd.DataFrame(columns=column_list)
+
+
+for i in range(days):
+    root = Tk()  # noqa
+    root.withdraw()
+    folder_selected = filedialog.askdirectory()
+    x = folder_selected + '/*'
```

Comment:
very very very unfortunate variable name ;-) also you might doom Windows folks.  Use `os.path.join` to produce legit paths.

Reply:
fixed :)